pkg/csi/service/common/commonco/utils.go

```diff
@@ -21,7 +21,6 @@ import (
 	"strings"
 
 	cnstypes "github.com/vmware/govmomi/cns/types"
-
```

Comment:
nit: This newline has been removed in few other files as well. Could you put it back?

Reply:
done.